product_docs/docs/edb_plus/41/06_command_summary.mdx

```diff
@@ -118,7 +115,7 @@ If the `COLUMN` command is followed by a column name, then the column name can b
 
 -  No other options
 -  `CLEAR`
--  Any combination of `FORMAT`, `HEADING`, and either `OFF` or `ON`
+-  Any combination of `FORMAT`, `HEADING,NEW_VALUE` and either `OFF` or `ON`
 
 `column`
```

Comment:
The `column` parameter/option was changed to `column_name` above.  Given that, either the subsequent references to the `column` parameter for the COLUMN command should be updated to refer to the parameter as `column_name` or `column_name` changed back to `column`.

Reply:
I just made this update in my most recent commit. Please double-check that I changed the correct references.